spring-orm/src/test/java/org/springframework/orm/jpa/JpaTransactionManagerTests.java

```diff
@@ -417,9 +418,9 @@ public void testPropagationSupportsAndRequiresNewAndEarlyAccess() {
 		assertThat(TransactionSynchronizationManager.isSynchronizationActive()).isFalse();
 
 		Object result = tt.execute(status -> {
-			EntityManagerFactoryUtils.getTransactionalEntityManager(factory);
+			assertThat(EntityManagerFactoryUtils.getTransactionalEntityManager(factory)).isNull();
```

Comment:
PROPAGATION_SUPPORTS says, that if there is no existing transaction, the code will be executed non-transactionally, and, thus it makes sense for get Transactional EntityManager to return null.

That also means no EM resource is bound to the thread via the factory.